oe_content.post_update.php

```diff
@@ -129,3 +130,24 @@ function oe_content_post_update_00004(): void {
     }
   }
 }
+
+/**
+ * Change widget for "Event type" field to skos select in form display.
+ */
+function oe_content_post_update_00005() {
+  if (!\Drupal::moduleHandler()->moduleExists('oe_content_event')) {
+    return 'Module oe_content_event is not enabled.';
+  }
```

Comment:
Can't we just move this in `oe_content_event`? So it's ran only if that module is enabled?

Reply:
Normally we prefer to place upgrade paths to the main module to ensure order. I thought I keep consistency.

Reply:
This code belongs to `oe_content_event`, which depends on `oe_content`, we have no order issues here.